qiskit/quantum_info/operators/symplectic/pauli_list.py

```diff
@@ -903,18 +903,20 @@ def _commutes_with_all(self, other, anti=False):
     def evolve(self, other, qargs=None, frame="h"):
         r"""Evolve the Pauli by a Clifford.
 
-        This returns the Pauli :math:`P^\prime = C.P.C^\dagger`.
+        This returns the Pauli :math:`P^\prime = C^\dagger.P.C`.
 
-        By choosing the parameter frame='s', this function returns the Schrödinger evolution of the Pauli
+        By choosing the parameter ``frame='s'``,
+        this function returns the Schrödinger evolution of the Pauli
         :math:`P^\prime = C.P.C^\dagger`. This option yields a faster calculation.
 
         Args:
             other (Pauli or Clifford or QuantumCircuit): The Clifford operator to evolve by.
             qargs (list): a list of qubits to apply the Clifford to.
-            frame (string): 'h' for Heisenberg or 's' for Schrödinger framework.
+            frame (string): ``'h'`` for Heisenberg (default) or ``'s'`` for Schrödinger framework.
 
         Returns:
-            Pauli: the Pauli :math:`C.P.C^\dagger`.
+            Pauli: the Pauli :math:`C^\dagger.P.C` (Schrödinger framework)
+            or the Pauli :math:`C.P.C^\dagger` (Heisenberg framework).
```

Comment:
I never remember which way round the Heisenberg and Schrödinger pictures are, but this text currently seems to disagree with itself.  Lines 909--910 say the Schrödinger frame is $P' = CPC^\dagger$, but the return value says that the Schrödinger frame returns $C^\dagger PC$.

I 100% defer to Shelley and Ikko on which is which, but right now it reads to me that we're inconsistent.

Reply:
I think that it was mixed in this comment (and should be the other way round):
https://github.com/Qiskit/qiskit-terra/pull/10219#discussion_r1221264651

Reply:
Hi. I rebased my code with main and now my MR is showing more file changes than expected. Can you help me fix this or shall I just create a new MR?